pulpcore/cli/common/generic.py

```diff
@@ -328,74 +328,59 @@ def _version_callback(
     return value
 
 
-# TODO: would be great to have enable this to take a validator, rather than having
-# to build "on top of" it like I'm doing now w/ json_callback
 def load_file_or_string_callback(
```

Comment:
I'm not grocking the whole change yet, but this thing now looks deceivingly like a decorator. Do you think we can use it as one?

Reply:
So this one can be used as a callback or chained when used as a decorator...
Maybe that should be in the docstring.